PostProcessing/Runtime/Utils/RuntimeUtilities.cs

```diff
@@ -424,6 +427,9 @@ public static Matrix4x4 GenerateJitteredProjectionMatrixFromOriginal(PostProcess
             var jitteredMatrix = Matrix4x4.Frustum(planes);
 
             return jitteredMatrix;
+#endif
+
```

Comment:
I'll re-add the explicit math to jitter the matrices in order to support pre-2017.2.